app/src/test/java/com/mapzen/erasermap/model/ManifestDownLoaderTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.mapzen.erasermap.model
+
+import com.mapzen.erasermap.BuildConfig
+import com.mapzen.erasermap.PrivateMapsTestRunner
+import com.mapzen.erasermap.model.ManifestDownLoader
+
+import com.squareup.okhttp.mockwebserver.MockResponse
+import com.squareup.okhttp.mockwebserver.MockWebServer
+import com.squareup.okhttp.mockwebserver.RecordedRequest
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.annotation.Config
+
+import org.assertj.core.api.Assertions.assertThat
+import java.util.concurrent.TimeUnit
+
+RunWith(PrivateMapsTestRunner::class)
+Config(constants = BuildConfig::class, sdk=intArrayOf(21))
+public class ManifestDownLoaderTest {
+    var downLoader: ManifestDownLoader? = null
+    var server: MockWebServer? = null
+    var sampleResponse: String = "{\"minVersion\": 0.1,\r\n" +
+            "    \"vectorTileApiKeyReleaseProp\": \"vectorKey\",\r\n " +
+            "   \"valhallaApiKey\": \"routeKey\",\r\n    " +
+            "\"mintApiKey\": \"mintKey\",\r\n    " +
+            "\"peliasApiKey\": \"peliasKey\"}\r\n"
+
+    Before
+    throws(Exception::class)
+    public fun setup() {
+        downLoader = ManifestDownLoader()
+        server = MockWebServer()
+        server?.play()
+       downLoader?.host = server?.getUrl("/").toString()
+
+    }
+
+    After
+    throws(Exception::class)
+    public fun teardown() {
+        server?.shutdown()
+    }
+
+    Test
```

Comment:
Wow the `@` symbol is optional for JUnit annotations... weird. Nice job getting tests to run in Kotlin.